### ISSUE TITLE
[all] pushed storage objects need checked and periodically cleared of old items.

### ISSUE THREAD

**Issue by agentlame** [creesch/reddit-moderator-toolbox#219]
This is almost a note to myself, because I keep forgetting about it.  Notifier.pushed[thing] will currently add thing IDs forever.  (also, MMP.replied, and maybe some others)... This is not only bad behaviour, it could cause users to hit storage limits in a lot of weird ways (browser storage limits, and wiki storage limits with the new import/export).

All storage objects that behave this way need a check. Pseudocode example:

TBUtils.init() {
    if (getSettings('Module', 'pushedvar').length > 250) {
         getSettings('Module', 'pushedvar').splice(100, -1)  //remove the oldest 100 items.
    }
}

Basically, anything that pushes items need a higher bounds check and clearing.
